### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by mremi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,51 @@ capistrano-copy-files
...
+Require the module in your `Capfile`:
+
+```ruby
+require 'capistrano/copy_files'
+```
+
+The `deploy:copy_files` task will run during `deploy:updating` in order to copy
+files and/or directories from previous release. This allows to speed up a
+deploy: for instance, you can copy a vendor directory where dependencies have
+been previously installed by Composer.
```
Comment (peterjmit) [inline README.md]: This is a generic package so the following would be more appropriate.

> previously installed by a package manager such as composer or NPM.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,51 @@ capistrano-copy-files
...
+
+### Configuration
+
+Configurable options, shown here with defaults:
+
+```ruby
+set :copy_files, []
+set :copy_file_flags, ""
+set :copy_dir_flags, ""
```
Comment (peterjmit) [inline README.md]: This parameter comes with default of `-R`

**Comment by peterjmit** [comment]
@mremi Thank you so much for this!

**Comment by mremi** [comment]
@peterjmit updated ;-)

**Comment by mremi** [comment]
Ping @peterjmit , good for you?

**Comment by peterjmit** [comment]
@mremi yes, thanks again :)

**Pull request merged**